Kiosk/Views/SettingsPage.xaml

```diff
@@ -200,9 +214,25 @@
                     <ComboBox Margin="0,6,0,0" x:Name="cameraSourceComboBox" Header="Camera Source"  SelectionChanged="OnCameraSourceSelectionChanged"/>
                     <ToggleSwitch IsOn="{Binding ShowDebugInfo, Mode=TwoWay}" Header="Show debug info:" Margin="0,6,0,0"/>
 
-                    <StackPanel Orientation="Horizontal">
-                        <Slider x:Name="MinDetectableFaceSizeSlider" Minimum="0" Maximum="100" Value="{Binding MinDetectableFaceCoveragePercentage, Mode=TwoWay}" SmallChange="1" LargeChange="5" Header="Minimum detectable face size (as % of image height):" HorizontalAlignment="Left" Margin="0,0,0,0" StepFrequency="1"/>
-                        <TextBlock Text="{Binding ElementName=MinDetectableFaceSizeSlider, Path=Value}" Style="{StaticResource TitleTextBlockStyle}" VerticalAlignment="Center" Margin="12,0,0,0"/>
+                    <StackPanel Style="{StaticResource Settings.SubPanel}">
+                        <TextBlock Text="Face Settings" Style="{StaticResource Settings.SubTitle}"/>
+
+                        <StackPanel Orientation="Horizontal">
+                            <Slider x:Name="MinDetectableFaceSizeSlider" Minimum="0" Maximum="100" Value="{Binding MinDetectableFaceCoveragePercentage, Mode=TwoWay}" SmallChange="1" LargeChange="5" Header="Minimum detectable face size (as % of image height):" HorizontalAlignment="Left" Margin="0,0,0,0" StepFrequency="1"/>
+                            <TextBlock Text="{Binding ElementName=MinDetectableFaceSizeSlider, Path=Value}" Style="{StaticResource TitleTextBlockStyle}" VerticalAlignment="Center" Margin="12,0,0,0"/>
+                        </StackPanel>
+
+                        <ToggleSwitch Header="Age and gender prediction" IsOn="{Binding ShowAgeAndGender, Mode=TwoWay}">
+                            <ToggleSwitch.HeaderTemplate>
+                                <DataTemplate>
+                                    <StackPanel>
+                                        <TextBlock Text="Age and gender prediction"/>
+                                        <TextBlock Style="{StaticResource Settings.Comment}" Text="Predictions based on face appearance may not match a person's identity.&#x0a;Consider if seeing these estimates is appropriate for your audience.&#x0a;Please read the FAQ in the Kiosk User Guide for more." />
```

Comment:
Since we don't have a user guide here, let's remove the last part of this line (i.e. "&#x0a;Please read the FAQ in the Kiosk User Guide for more.")

Reply:
Fixed.